content/docs/hooks-state.md

```diff
@@ -28,11 +29,11 @@ function Example() {
 }
 ```
 
-We'll start learning about Hooks by comparing this code to an equivalent class example.
+נתחיל ללמוד על Hooks על ידי השווה בין קוד זה לקוד המקביל שמשתמש במחלקה.
 
-## Equivalent Class Example {#equivalent-class-example}
+## דוגמת קוד מקביל {#equivalent-class-example}
```

Comment:
What do you think about "דוגמה מקבילה במחלקה", I just try to stick to the original.

Reply:
You might be right but imo "דוגמה מקבילה במחלקה" doesn't fit